scripts/cue.sh

```diff
@@ -5,7 +5,8 @@
 set -e
 
 function fmt() {
-  find ./internal ./pkg ./schemas -name "*.cue" -exec cue fmt {} \;
+  # exclude migration files from the formatting since it removes the trailing commas we need
+  find ./internal ./pkg ./schemas -name "*.cue" -and -not -name "migrate.cue" -exec cue fmt {} \;
```

Comment:
can't we add it programmatically and keep the formatter on every cue file ?

Reply:
the problem is that it's not as simple as adding a comma at the end of the payload: you can have several conditional blocks in a same file (like https://github.com/perses/perses/pull/1489/files#diff-15277e993e33a4adf4c22a6d46eed9f9582f2b0906824f999cf6a45d2b8cdb03)